telemetry/main.schema.json

```diff
@@ -19,7 +19,9 @@
             "nightly",
             "aurora",
             "beta",
-            "release"
+            "release",
+            "esr",
+            "default"
```

Comment:
`channel` should just be a string, I don't think we want to specifically enumerate the list of possible values (there are a lot of variations in the long tail, such as `release-cck-yahoo`).